src/e2e_test/perf_tool/pkg/run/run.go

```diff
@@ -55,10 +55,14 @@ type Runner struct {
 	clusterCtx          *cluster.Context
 	clusterCleanup      func()
 	vizier              deploy.Workload
-	wg                  sync.WaitGroup
 	metricsResultCh     chan *metrics.ResultRow
 	metricsBySelector   map[string][]metrics.Recorder
 	workloadsBySelector map[string][]deploy.Workload
+
+	// wg is for goroutines that are unrelated to the main execution of the experiment.
+	wg sync.WaitGroup
```

Comment:
don't they also include the experiment? seems like wg also includes stuff in eg.

Reply:
The only thing the wait group is used for at the moment is the bigquery inserter which runs in the background and moves results from the result channel to bigquery.